Drop after FreeBSD 12.4 EOL around 2023-12-31.

Utilities/Config.cpp:194:23: error: no matching function for call to 'to_chars'
        if (auto [ptr, ec] = std::to_chars(str.data(), str.data() + str.size(), value, std::chars_format::fixed); ec == std::errc())
                             ^~~~~~~~~~~~~
/usr/include/c++/v1/charconv:166:6: note: candidate function not viable: no known conversion from 'std::chars_format' to 'int' for 4th argument
void to_chars(char*, char*, bool, int = 10) = delete;
     ^
/usr/include/c++/v1/charconv:458:1: note: candidate template ignored: requirement 'is_integral<double>::value' was not satisfied [with _Tp = double]
to_chars(char* __first, char* __last, _Tp __value, int __base)
^
/usr/include/c++/v1/charconv:450:1: note: candidate function template not viable: requires 3 arguments, but 4 were provided
to_chars(char* __first, char* __last, _Tp __value)
^

--- Utilities/Config.cpp.orig	2023-02-25 10:23:07 UTC
+++ Utilities/Config.cpp
@@ -185,7 +185,7 @@ bool try_to_string(std::string* out, const f64& value)
 
 bool try_to_string(std::string* out, const f64& value)
 {
-#ifdef __APPLE__
+#if defined(_LIBCPP_VERSION) && _LIBCPP_VERSION < 14000
 	if (out) *out = std::to_string(value);
 	return true;
 #else
